install_config/topics/manually_configuring_for_gce.adoc

```diff
@@ -0,0 +1,126 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_gce.adoc
+////
+
+[id='gce-configuring-masters-manual_{context}']
+= Option 2: Manually configuring {product-title} for GCE
+
+== Manually configuring master hosts for GCE
+
+Perform the following on all master hosts.
+
+.Procedure
+
+. Edit the master configuration file at `/etc/origin/master/master-config.yaml`
```

Comment:
I'd say something like "Add the GCE parameters to the `apiServerArguments` and `controllerArguments` sections of the master configuration file, which is located in the `...` directory: